pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -1265,17 +1265,17 @@ func (r *ClickHouseReader) GetServiceOverview(ctx context.Context, queryParams *
 
 	if err != nil {
 		zap.S().Debug("Error in processing sql query: ", err)
-		return nil, &model.ApiError{Typ: model.ErrorExec, Err: fmt.Errorf("Error in processing sql query")}
+		return nil, &model.ApiError{Typ: model.ErrorExec, Err: fmt.Errorf("error in processing sql query")}
 	}
 
 	m := make(map[int64]int)
 
 	for j := range serviceErrorItems {
-		m[int64(serviceErrorItems[j].Time.UnixNano())] = int(serviceErrorItems[j].NumErrors)
+		m[serviceErrorItems[j].Time.UnixNano()] = int(serviceErrorItems[j].NumErrors)
```

Comment:
was this auto-suggested? Please double check else this can result in errors

Reply:
Yes, I checked. UnixNano already returns int64 https://pkg.go.dev/time#Time.UnixNano